### DISCUSSION TITLE
Why only beta releases?

### DISCUSSION THREAD

**Discussion by janos-r** [doublecmd/doublecmd#1395] [General]
This way the release doesn't reach
`http://download.opensuse.org/repositories/home:/Alexx2000/xUbuntu_22.04`
I think that is the reason I don't get it suggested and I would like to keep it installed where the package manager put it.
Please make normal releases again if there is not a stability issue to keep them back.

**Comment by alexx2000** [comment]
Because real release is not ready yet.
